spec/chromium-spec.js

```diff
@@ -1012,8 +1034,8 @@ describe('chromium feature', () => {
       w.webContents.loadURL(pdfSource)
     })
 
-    it('should not open when pdf is requested as sub resource', (done) => {
-      createBrowserWindow({plugins: true})
+    it('should not open when pdf is requested as sub resource', function (done) {
```

Comment:
Why `(done) =>` was changed to `function (done)`?

Reply:
@alexeykuzmin,
```function (done)``` was removed after my fork with the following change
[WIP] Upgrade more specs (#10945)
https://github.com/electron/electron/commit/e4214a6cbe084f6e038d955ae0a61df9394cd486

I think I have also missed it during the conflict resolution. Will fix it.